scripts/setup-operators.sh

```diff
@@ -2,21 +2,53 @@
 set -x
 
 CI_OPERATOR_HUB_PROJECT=ci-operator-hub-project
-# First, enable a cluster-wide mongo operator
-oc create -f - <<EOF
-apiVersion: operators.coreos.com/v1alpha1
-kind: Subscription
-metadata:
-  generation: 1
-  name: mongodb-enterprise
-  namespace: openshift-operators
-spec:
-  channel: stable
-  installPlanApproval: Automatic
-  name: mongodb-enterprise
-  source: certified-operators
-  sourceNamespace: openshift-marketplace
+
+install_mongo_operator() {
+  # First, enable a cluster-wide mongo operator
+  oc create -f - <<EOF
+  apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    generation: 1
+    name: mongodb-enterprise
+    namespace: openshift-operators
+  spec:
+    channel: stable
+    installPlanApproval: Automatic
+    name: mongodb-enterprise
+    source: certified-operators
+    sourceNamespace: openshift-marketplace
+EOF
+}
+
+install_etcd_operator(){
+  # Create subscription
+  oc create -f - <<EOF
+  apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    name: etcd
+    namespace: ${OPERATOR_HUB_PROJECT}
+  spec:
+    channel: singlenamespace-alpha
+    installPlanApproval: Automatic
+    name: etcd
+    source: community-operators
+    sourceNamespace: openshift-marketplace
 EOF
+}
+
+# install mongo operator
+while :
```

Comment:
We should limit this, for example try max of 3 times otherwise there is a high chance of infinite loop or consume a lots of time until it finds the proper grep line.